convert2rhel/pkghandler.py

```diff
@@ -291,6 +292,7 @@ def list_non_red_hat_pkgs_left():
         print_pkg_info(non_red_hat_pkgs)
     else:
         loggerinst.info("All packages are now signed by Red Hat.")
+    return
```

Comment:
It is unnecessary `return` at the end of the function. Anyway, the function returns `None`

Reply:
Yeah..  we can start removing these kind of return of many function we have...